### PR TITLE
FinancialInstitutionTransactionsService.update

### PR DESCRIPTION
**Pull request by cleclefibanity**

### CODE REVIEW

**Comment by damienbr** [comment]
:+1: I guess we need to wait for the release before merging

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/UpdatedTransactionsServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+
+        String url = getUrl(updatedTransactionsReadQuery.getSynchronizationId());
+        HttpResponse response = ibanityHttpClient.get(buildUri(url, pagingSpec), updatedTransactionsReadQuery.getAdditionalHeaders(), updatedTransactionsReadQuery.getCustomerAccessToken());
+
+        return mapCollection(response, Transaction.class);
+    }
+
+    private String getUrl(UUID synchronizationId) {
+        String url = apiUrlProvider.find(IbanityProduct.Xs2a, "customer", "synchronizations", "updatedTransactions");
```
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/UpdatedTransactionsServiceImpl.java:44]: synchronizations -> synchronization

File: src/main/java/com/ibanity/apis/client/products/xs2a/mappers/SynchronizationMapper.java [hunk 2]
```diff
@@ -30,6 +31,13 @@ public static Synchronization map(DataApiModel dataApiModel) {
         if (dataApiModel.getLinks() != null) {
             result.setSelfLink(dataApiModel.getLinks().getSelf());
         }
+        if (dataApiModel.getRelationships() != null
```
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/mappers/SynchronizationMapper.java:34]: if you update src/test/resources/json/synchronization.json, you can make sure that the parsing works properly in SyncrhonizationServiceImplTest.java

**Review by damienbr — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -331,7 +331,7 @@
                 </configuration>
             </plugin>
             <!-- To sign the artifacts -->
-            <plugin>
+            <!--<plugin>
```
Comment (damienbr) [inline pom.xml:334]: :grimacing:

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java [hunk 4]
```diff
@@ -37,7 +38,7 @@ public TransactionsServiceImpl(ApiUrlProvider apiUrlProvider, IbanityHttpClient
             pagingSpec = IbanityPagingSpec.DEFAULT_PAGING_SPEC;
         }
 
-        String url = getUrl(transactionsReadQuery.getFinancialInstitutionId(), transactionsReadQuery.getAccountId());
+        String url = getUrl(transactionsReadQuery.getFinancialInstitutionId(), transactionsReadQuery.getAccountId(), transactionsReadQuery.getSynchronizationId());
```
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java:41]: Maybe you could add a test in TransactionsServiceImplTest -> @Test listUpdatedTransactions, in order to make sure we don't break this behavior in the future?

**Review by damienbr — APPROVED** [review]

**Review by damienbr — CHANGES_REQUESTED** [review]
Rollback of the pom required.

File: pom.xml (hunk 3 above)
Comment (cleclefibanity) [reply pom.xml:334]: oups :-) Done

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java (hunk 4 above)
Comment (cleclefibanity) [reply src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java:41]: added

**Review by damienbr — APPROVED** [review]

**Pull request merged**